vendor/bat-native-ads/src/bat/ads/internal/database/tables/transactions_database_table.cc

```diff
@@ -172,9 +172,10 @@ void Transactions::Update(
       "UPDATE %s "
       "SET reconciled_at = %s "
       "WHERE reconciled_at == 0 "
-      "AND id IN %s",
+      "AND id IN %s OR creative_instance_id IN %s",
```

Comment:
There could be an error connected with boolean operators precedence. Should it be:
`"AND (id IN %s OR creative_instance_id IN %s)",`

Reply:
Fixed